app/src/main/java/com/vinh/dictionary_1/data/source/local/evdictdatabase/EVDictLocalDatasource.java

```diff
@@ -1,9 +1,14 @@
 package com.vinh.dictionary_1.data.source.local.evdictdatabase;
 
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports